pkg/config/args.go

```diff
@@ -51,3 +53,32 @@ func (b *multiArg) Contains(v string) bool {
 	}
 	return false
 }
+
+// This type is used to supported passing in multiple key=value flags
+type keyValueArg map[string]string
+
+// Now, for our new type, implement the two methods of
+// the flag.Value interface...
+// The first method is String() string
+func (a *keyValueArg) String() string {
+	var result []string
+	for key := range *a {
+		result = append(result, fmt.Sprintf("%s=%s", key, (*a)[key]))
+	}
+	return strings.Join(result, ",")
+}
+
+// The second method is Set(value string) error
+func (a *keyValueArg) Set(value string) error {
+	valueSplitted := strings.Split(value, "=")
```

Comment:
What if the value contains `=`? Sorry to nit pick on the some conjugation.

```suggestion
	valueSplit := strings.SplitN(value, "=", 2)
```

Reply:
@greut thanks. I made the change and wrote some unit tests specifying  this behaviour.